### PR TITLE
(WIP) Code cleanup, setup travis-ci

### PR DESCRIPTION
**Pull request by coreydaley**
Cleaning up code in the import_content.py script
Adding .travis.yml to test for:
 - Python errors (pylint)
 - diff official and community directories for pull requests

### CODE REVIEW

**Comment by bparees** [comment]
still not seeing the new laravel files?

**Comment by coreydaley** [comment]
@bparees ptal

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+#
+
+# Run core verification and all self contained tests.
+#
+# Example:
+#   make verify
+verify:
+	hack/verify-python.sh
+	hack/verify-pullrequest.sh
+.PHONY: verify
```
Comment (bparees) [inline Makefile]: how about a make target that actually does the import?

Comment (coreydaley) [inline Makefile]: Updated in next push

**Comment by bparees** [comment]
lgtm

**Pull request merged**